src/Template/Element/Flash/default.ctp

```diff
@@ -0,0 +1,11 @@
+<?php
+$class = 'alert';
+if (!empty($params['class'])) {
```

Comment:
Using array of classes does make more sense. 

Regarding making an alert dismissable how about we add an option `dismissable` (default true) to the FlashHelper?  If true the helper can automatically add `alert-dismissable` to the classes list. This will help keep controller code DRY.